release-deployment.md

```diff
@@ -113,7 +116,7 @@ There's nothing special about that. Each developer follows the above [Develop a
    * Base: `master`
    * Compare: `release-vX.Y.Z`
    Paste the Release Checklist into the PR body. Each project should define a release
-   checklist. It will vary across projects, but you can refer to the Astro [Release](https://github.com/mobify/astro/blob/develop/RELEASE.md) document
+   checklist. It will vary across projects, but you can refer to the Mobify Code Style [Release](https://github.com/mobify/mobify-code-style/blob/develop/RELEASE.md) document
```

Comment:
Minor point, I'd try to always reference `master` branches, as a best practice to try and reference reliable/production content.